onnxruntime/server/server_configuration.h

```diff
@@ -97,9 +97,6 @@ class ServerConfiguration {
     } else if (num_http_threads <= 0) {
       PrintHelp(std::cerr, "num_http_threads must be greater than 0");
       return Result::ExitFailure;
-    } else if (http_port < 0 || http_port > 65535) {
```

Comment:
Why remove it?

Reply:
unsigned shorts are only between 0 and 65535, this check fails on Windows because it's impossible to ever be true